src/getTableDefinition.ts

```diff
@@ -5,6 +5,10 @@ export interface TableDefinition {
   [columnName: string]: ColumnDefinition;
 }
 
+export interface MultiTableDefinition {
+  [tableName: string]: TableDefinition;
+}
+
```

Comment:
I would not create a new type for this, just set the return type of getTablesDefinition to `Promise<Record<string, TableDefinition>>`

Reply:
Done